Avoid export default (#71)
wwnorton/style: guides-website/docs/javascript-style.md

export { foo };
```

### Named and Default Export

There are two different types of export, named and default. You can have multiple named exports per module but only one default export.

> In JS modules with a single export, prefer default export over named export. As for Typescript we want to prefer named export and avoide default export. See the [Typescript docs](https://wwnorton.github.io/style/docs/typescript#import--export) for more info.

Reviewer: ```suggestion
In JS modules with a single export, prefer default export over named export.

> Note: this is inverted in Typescript, where we [prefer named exports and avoid default export](https://wwnorton.github.io/style/docs/typescript#import--export).
```

eslint: [`import/prefer-default-export`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md)

> > Why? To encourage more files that only ever export one thing, which is better for readability and maintainability. However, named exports provide consistency throughout all files with a focus on being explicit over implicit. TypeScript can also leverage it's auto complete and auto suggest features with named exports.

Reviewer: I think you should revert this. If someone's reading about JavaScript style, info about TypeScript style could feel extraneous at best and confusing at worst.
